assets/xml/objects/object_fish.xml

```diff
@@ -1,162 +1,315 @@
-        <Limb Name="object_fish_Standardlimb_00293C" Type="Standard" EnumName="FISHING_FISH_LIMB_0A" Offset="0x293C" />
-        <Limb Name="object_fish_Standardlimb_002948" Type="Standard" EnumName="FISHING_FISH_LIMB_0B" Offset="0x2948" />
-        <Limb Name="object_fish_Standardlimb_002954" Type="Standard" EnumName="FISHING_FISH_LIMB_0C" Offset="0x2954" />
-        <Limb Name="object_fish_Standardlimb_002960" Type="Standard" EnumName="FISHING_FISH_LIMB_0D" Offset="0x2960" />
-        <Limb Name="object_fish_Standardlimb_00296C" Type="Standard" EnumName="FISHING_FISH_LIMB_0E" Offset="0x296C" />
-        <Limb Name="object_fish_Standardlimb_002978" Type="Standard" EnumName="FISHING_FISH_LIMB_0F" Offset="0x2978" />
-        <Skeleton Name="gFishingFishSkel" Type="Flex" LimbType="Standard" LimbNone="FISHING_FISH_LIMB_NONE" LimbMax="FISHING_FISH_LIMB_MAX" EnumName="FishingFishLimb" Offset="0x29C0" />
-        <Texture Name="object_fish_Tex_0029D0" OutName="tex_0029D0" Format="rgba16" Width="8" Height="16" Offset="0x29D0" />
-        <Texture Name="object_fish_Tex_002AD0" OutName="tex_002AD0" Format="rgba16" Width="8" Height="8" Offset="0x2AD0" />
-        <DList Name="object_fish_DL_002BD0" Offset="0x2BD0" />
-        <Texture Name="object_fish_Tex_002CE0" OutName="tex_002CE0" Format="i4" Width="32" Height="64" Offset="0x2CE0" />
-        <Texture Name="object_fish_Tex_0030E0" OutName="tex_0030E0" Format="i4" Width="16" Height="16" Offset="0x30E0" />
+
+        <!-- Bass -->
+
+        <!-- Bass Animations -->
+        <Animation Name="gFishingBassAnim" Offset="0x7C" /> <!-- Original name is "basu" -->
+
+        <!-- Bass DLs -->
+        <DList Name="gFishingBassTailFinDL" Offset="0x940" />
+        <DList Name="gFishingBassBackSegmentEndDL" Offset="0x9E8" />
+        <DList Name="gFishingBassTopRearFinDL" Offset="0xB00" />
+        <DList Name="gFishingBassBottomRearFinDL" Offset="0xBA0" />
+        <DList Name="gFishingBassBackSegmentDL" Offset="0xC40" />
+        <DList Name="gFishingBassTopFrontFinDL" Offset="0xE30" />
+        <DList Name="gFishingBassBottomFrontFinDL" Offset="0xED0" />
+        <DList Name="gFishingBassMiddleSegmentDL" Offset="0xF70" />
+        <DList Name="gFishingBassHeadDL" Offset="0x10A0" />
+        <DList Name="gFishingBassLeftPectoralFinDL" Offset="0x1260" />
+        <DList Name="gFishingBassRightPectoralFinDL" Offset="0x1300" />
+        <DList Name="gFishingBassJawDL" Offset="0x13A0" />
+
+        <!-- Bass Textures -->
+        <Texture Name="gFishingBassTailFinTex" OutName="fishing_bass_caudal_fin" Format="rgba16" Width="16" Height="16" Offset="0x14D0" />
+        <Texture Name="gFishingBassTopScalesTex" OutName="fishing_bass_dorsal_scales" Format="rgba16" Width="16" Height="16" Offset="0x16D0" />
+        <Texture Name="gFishingBassBottomScalesTex" OutName="fishing_bass_belly_scales" Format="rgba16" Width="16" Height="16" Offset="0x18D0" />
```

Comment:
Could you update the `OutName`'s so that they match the texture names in snake case?